interfaces/handler/contest.go

```diff
@@ -225,6 +232,41 @@ func (h *ContestHandler) GetContestTeams(_c echo.Context) error {
 	return c.JSON(http.StatusOK, res)
 }
 
+// GetContestTeams GET /contests/:contestID/teams/:teamID
+func (h *ContestHandler) GetContestTeam(_c echo.Context) error {
+	c := Context{_c}
+	ctx := c.Request().Context()
+	_id := c.Param("contestID")
+	contestID := uuid.FromStringOrNil(_id)
+	_id = c.Param("teamID")
+	teamID := uuid.FromStringOrNil(_id)
+	contestTeam, err := h.srv.GetContestTeam(ctx, contestID, teamID)
+	if err != nil {
+		return err
+	}
+
+	members := make([]*userResponse, 0, len(contestTeam.Members))
+	for _, user := range contestTeam.Members {
+		members = append(members, &userResponse{
+			ID:       user.ID,
+			Name:     user.Name,
+			RealName: user.RealName,
+		})	
```

Comment:
括弧の後に空白があったのがダメみたいです🙏
```suggestion
		})
```